scenarios/arcaflow/kill-pod/workflow.yaml

```diff
@@ -0,0 +1,42 @@
+input:
```

Comment:
We might want to tweak the docs to walk through the workflow and how to tweak it as per user needs: https://github.com/redhat-chaos/krkn/blob/main/docs/pod_scenarios.md.